### ISSUE TITLE
Audio but no video for youtube in debian sid

### ISSUE THREAD

**Issue by Permafacture** [The-Compiler/qutebrowser#777]
Installed gstreamer libs, uninstalled and reinstalled qutebrowser, rebooted system, and still no video in youtube.

```
$ dpkg --get-selections | grep gstreamer
gstreamer0.10-gconf:amd64			install
gstreamer0.10-plugins-bad:amd64			install
gstreamer0.10-plugins-base:amd64		install
gstreamer0.10-plugins-good:amd64		install
gstreamer0.10-plugins-ugly:amd64		install
gstreamer0.10-x:amd64				install
gstreamer1.0-libav:amd64			install
gstreamer1.0-plugins-bad:amd64			install
gstreamer1.0-plugins-base:amd64			install
gstreamer1.0-plugins-good:amd64			install
gstreamer1.0-plugins-ugly:amd64			install
gstreamer1.0-x:amd64				install
libgstreamer-plugins-bad0.10-0:amd64		install
libgstreamer-plugins-bad1.0-0:amd64		install
libgstreamer-plugins-base0.10-0:amd64		install
libgstreamer-plugins-base1.0-0:amd64		install
libgstreamer0.10-0:amd64			install
libgstreamer1.0-0:amd64				install
```

**Comment by Permafacture** [comment]
The issue is only h.264 video (tested here:http://www.quirksmode.org/html5/tests/video.html) I think this is a debian sid thing.  gstreamer0.10-ffmpeg is not apt-get installable, and qtwebkit doesn't use gstreamer1.0 unless I build it...

**Comment by The-Compiler** [comment]
Thanks for the update! I was about to say, I'm a bit confused on why it wouldn't work...

Looking at the [Debian bug #756098](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=756098) it seems [libqtwebkit4](https://packages.debian.org/sid/libqtwebkit4) uses gstreamer1.0 while [libqt5webkit5](https://packages.debian.org/sid/libqt5webkit5) uses 0.10 (the 5.4.1 package in experimental does the same)... Weird. On Arch, I have QtWebKit with GStreamer 1.0 running without any issues.

I think you should open a Debian bug (e.g. using `reportbug libqt5webkit5`) and mention that there. Please link the bug here if you do.

**Comment by Permafacture** [comment]
I hope I didn't spam them, but I sent a report through reportbug and a second through email (when the first didn't show up) to an existing bug report: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=785862

They plan to fix this when they go from qtwebkit 5.4 to 5.5.

**Comment by perezmeyer** [comment]
Right, Qt5 will gain gstreamer1.0 support in 5.5.0, which is due for July 1st (hopefully).
